content/docs/fundamentals/converting/resolving-errors@id.md

```diff
@@ -0,0 +1,350 @@
+
+Untuk menyelesaikan error viewport, tambahkan cuplikan HTML berikut ke tag `<head>`:
+
+```html
+<meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+```
+
+Nilai untuk `width` dan `minimum-scale` wajib ditetapkan di AMP. Menetapkan `initial-scale` tidak wajib dilakukan, tapi biasanya disertakan dalam pengembangan web seluler dan hal itu direkomendasikan. Anda dapat membaca selengkapnya tentang viewport dan desain yang responsif di [Mengonfigurasi Area Pandang](https://developers.google.com/speed/docs/insights/ConfigureViewport).
+
+Seperti sebelumnya, **muat ulang** halaman dan periksa apakah error sudah tidak muncul.
+
+## Ganti stylesheet eksternal
+
+Error berikut berkaitan dengan penggunaan kita atas stylesheet:
+
+<pre class="error-text">
+Atribut 'href' di tag 'link rel=stylesheet for fonts' ditetapkan ke nilai yang tidak valid 'base.css'.
+</pre>
+
+Secara spesifik, error ini berkaitan dengan tag link stylesheet berikut di tag `<head>`:
+
+```html
+<link href="base.css" rel="stylesheet" />
+```
+
+Masalahnya, ini adalah referensi stylesheet eksternal. Di AMP, agar kecepatan waktu pemuatan dokumen tetap maksimal, Anda tidak dapat menyertakan stylesheet eksternal. Sebagai gantinya, semua aturan stylesheet harus ditambahkan secara inline dalam dokumen AMP menggunakan tag `<style amp-custom></style>`.
+
+```html
+<style amp-custom>
+
+/* Konten dari base.css */
+
+</style>
+```
+
+Mari kita selesaikan error-nya:
+
+1.  **Hapus** tag `<link>` yang mengarah ke stylesheet di `<head>` dan ganti dengan tag `<style amp-custom></style>` inline. Atribut `amp-custom` wajib disertakan di tag gaya.
+2. **Salin** semua gaya dari file [`base.css`](https://github.com/googlecodelabs/accelerated-mobile-pages-foundations/blob/master/base.css) ke tag `<style amp-custom></style>`.
+
+Sekali lagi, **muat ulang** halaman dan verifikasi bahwa error stylesheet sudah tidak muncul.
+
+{% call callout('Catatan', type='note') %}
+Tidak hanya gaya inline yang diperlukan tetapi ada batas ukuran file sebesar 50 kilobyte untuk semua informasi gaya. Anda harus menggunakan prapemrosesan CSS seperti [SASS](http://sass-lang.com/) untuk memperkecil CSS sebelum membuat CSS di halaman AMP menjadi inline.
+{% endcall %}
+
+{% call callout('Penting', type='caution') %}
+Anda hanya dapat memiliki 1 tag gaya di seluruh dokumen AMP. Jika Anda memiliki beberapa stylesheet eksternal yang dirujuk oleh halaman AMP, Anda harus menyusun stylesheet ini menjadi 1 kumpulan aturan. Untuk mempelajari tentang aturan CSS apa yang berlaku di AMP, baca [CSS yang Didukung](/id/docs/design/responsive/style_pages.html).
+{% endcall %}
+
+## Jangan sertakan JavaScript pihak ketiga
+
+Meskipun stylesheet dapat dikerjakan ulang dengan relatif mudah menggunakan AMP dengan membuat CSS menjadi inline, hal yang sama tidak berlaku untuk JavaScript.
+
+<pre class="error-text">
+Tag 'script' tidak diizinkan, kecuali dalam bentuk yang spesifik.
```

Comment:
All of these code blocks need to be in EN, so please update with EN translation (https://www.ampproject.org/docs/fundamentals/converting/resolving-errors).

Reply:
Updated code block with EN version.